File: baekjoon/js/2744.js
const fs = require("fs");
const buf = fs.readFileSync(0, "utf-8").toString().trim();

let result = "";

for (const c of buf) {
  if (c.toLowerCase() === c) {
    result += c.toUpperCase();
  } else {
    result += c.toLowerCase();
  }
}

console.log(result);

File: baekjoon/rust/2744.rs
use std::io::stdin;

fn main() {
    let mut buf = String::new();
    stdin().read_line(&mut buf).unwrap();

    let mut result = String::new();

    for c in buf.chars() {
        let converted_c = if c.to_lowercase().next().unwrap() == c {
            c.to_uppercase().next().unwrap()
        } else {
            c.to_lowercase().next().unwrap()
        };

        result.push(converted_c);
    }

    print!("{}", result);
}

fn main() {
    let mut buf = String::new();
    stdin().read_line(&mut buf).unwrap();

    let result: String = buf
        .chars()
        .map(|c| match c {
            'A'..='Z' => c.to_ascii_lowercase(),
            'a'..='z' => c.to_ascii_uppercase(),
            _ => c,
        })
        .collect();

    print!("{}", result);
}
